Habitica/src/main/java/com/habitrpg/android/habitica/ui/activities/FullProfileActivity.java

```diff
@@ -263,7 +263,6 @@ public void onClick(View view) {
 
         // Load the members achievements now
         apiHelper.apiService.GetMemberAchievements(this.userId)
-                .compose(apiHelper.configureApiCallObserver())
```

Comment:
removing this isn't the fix to the compilation problem. all API calls need to be configured, because that also defines that they shouldn't run on the main thread. otherwise there is an error, when the api call is actually used.
I pushed a fix to develop, that fixes the original issue, which is that the GetMemberAchievements API call wasn't wrapped in HabitResponse, which is required since the login incentive PR.